containerized_ml_models/model_api_10/api.py

```diff
@@ -0,0 +1,24 @@
+import json
+import pickle
+from flask import Flask, request
+
+app = Flask(__name__)
+
+with open("model.pkl", "rb") as file:
```

Comment:
In general, it is better to keep paths string in variable in separate file (it is easier to handle changes if projects become bigger).